hac_nav.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: cloud.redhat.com/v1alpha1
+kind: Bundle
+metadata:
+  name: hac-navigation
+spec:
+  id: hac
+  envName: env-boot
+  title: HAC
+  extraNavItems:
```

Comment:
Mhh, we shouldn't be needing this. It should be enough just to add
```yaml
appList:
    - hac-core
```

And change how the navigation is generated in `hac.yml`